docs/development_guide.md

```diff
@@ -88,6 +142,15 @@ For other projects you can use the `--project` and `--gs_dir` options of the
 script.
 
 ### Pushing changes to your fork's branch
+
+Before pushing changes, make sure the pylint checks pass. To install pylint:
+```bash
+sudo apt-get install pylint
+```
+Then run:
+```bash
+pylint gcp_variant_transforms
```

Comment:
Hmm, I think you should use the `.pylintrc` file that is checked into the git repo not the default RC file. I usually run `pylint --rcfile=.pylintrc gcp_variant_transforms/`

Reply:
Sorry, I tried both and it seems that they are doing the same checking. Does it mean that the default file is .pylintrc?

Reply:
ditto